src/_langs/en/tools/javascript/command-line/events.markdown

```diff
@@ -3,40 +3,93 @@ rss: false
 layout: article
 title: "Monitor Events"
 seotitle: "Monitor Events Using the Chrome DevTools Command Line API"
-description: "TBD description."
-introduction: "TBD introduction."
+description: "The Chrome DevTools Command Line API offers various ways to observe and inspect Event Listeners"
+introduction: "The Chrome DevTools Command Line API offers various ways to observe and inspect Event Listeners"
 article:
   written_on: 2015-04-14
-  updated_on: 2015-04-14
+  updated_on: 2015-05-11
   order: 2
 authors:
   - megginkearney
+  - flaviocopes
 priority: 0
 collection: command-line
 key-takeaways:
-  tldr-tbd:
-    - TBD tldr.
-remember:
-  note-tbd:
-    - TBD note.
+  tldr:
+    - Use monitorEvents() to listen to events of a certain type
+    - Use unmonitorEvents() to stop listening
+    - Use getEventListeners() to get the event listeners of a DOM element
+    - Use the Event Listeners Inspector panel to get information on event listeners
 ---
 {% wrap content %}
 
-TBD. Specified parts of this doc: https://developer.chrome.com/devtools/docs/console
 
-TBD. Includes:
+# Monitor Events
 
-** Monitoring events
-** View event listeners registered on objects
+JavaScript plays a central role in interactive pages, and the Browser can provides you some useful tools to debug events and event handlers.
 
-{% include modules/takeaway.liquid list=page.key-takeaways.tldr-tbd %}
+## Monitoring events
```

Comment:
Do you need a monitor events and then a monitoring events subsection? Can we make it into one? I think this makes way more sense.